metadata/health/health_care_districts.md

```diff
@@ -0,0 +1,74 @@
+# Title
+
+Utah Health Care Districts
+
+## ID
+
+8bd90a3f-5920-401f-b1e5-a2691b93cacf
+
+## Brief Summary
+
+Polygon dataset of districts corresponding to local health department boundaries.
+
+## Summary
+
+This dataset contains polygons representing health care districts in Utah. This layer contains boundary information only and does not contain office locations or contact information for local health departments. Further health-related data, such as [Licensed Health Care Facilities](https://gis.utah.gov/products/sgid/health/licensed-health-care-facilities/), can be found in the [Health Data Index](https://gis.utah.gov/products/sgid/health/) available from the SGID.
+
+## Description
+
+### What is the dataset?
+
+A health care district is composed of one or several counties that are grouped together for public health administration purposes. Health care districts are managed by a local or regional Department of Health and assist the communities in their jurisdiction with clinical services, emergency preparedness, and data collection on diseases and other health concerns. There are 12 health care districts in Utah, six of which contain singular counties (Davis, Salt Lake, Summit, Tooele, Utah, and Wasatch).
+
+### What is the purpose of the dataset?
+
+This dataset is used by a variety of local and state agencies for statistical analysis and regional public health planning.
+
+<!--- This is my assumption. Do we know if these data have a more specific purpose than this? --->
```

Comment:
I can't think of anything. There may be something in code regarding the purposes of the local health departments themselves, but anytime I've seen the layer used is for mapping or analysis.